.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+#### Please paste your output of croutonversion from chroot below:
+```
+Replace this line with your output.
```

Comment:
I also added this bit of markdown syntax for better format. Hopefully people will not mess the Markdown syntax up.